test/support/helpers.ex

```diff
@@ -10,6 +10,6 @@ defmodule EpiLocator.Test.Helpers do
   def read_tr_xml_and_turn_into_map(filename) do
     filename
     |> read_xml_and_turn_into_map()
-    |> Map.get("{http://clear.thomsonreuters.com/api/search/2.0}PersonResultsPage")
+    |> Map.get("{http://clear.thomsonreuters.com/api/search/2.0}PersonResultsPageV3")
```

Comment:
`/api/v3` ... `/api/search/2.0` ... cool

(kind of a weird naming / versioning scheme they've got going on imo)